container-scan/action.yaml

```diff
@@ -1,152 +1,78 @@
-name: "Security Scan Composite Action"
-description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+name: "Container Security Scan"
+description: "Scans container images for vulnerabilities using Lacework"
 
 inputs:
-  dockerhub-user:
+  image-name:
     required: false
-    default: ""
-    description: username for dockerhub
-  dockerhub-password:
+    description: "Docker image name"
+  image-tag:
+    required: true
+    description: "Docker image tag"
+  enable-docker-build:
     required: false
-    default: ""
-    description: password for dockerhub
-  docker-config-file:
+    default: "true"
+    description: "Enable Docker build"
+  build-args:
     required: false
-    description: Path to the docker config file (defaults to .docker-config.json) Must contain imageName, may contain dockerfile
-    default: .docker-config.json
-  github-token:
-    required: true
-    description: GitHub token
+    description: "Docker build arguments"
   lw-account-name:
     required: true
-    description: Lacework account name
+    description: "Lacework account name"
   lw-access-token:
     required: true
-    description: Lacework access token
-  image-name:
-    required: false
-    description: Docker image name
-  image-tag:
+    description: "Lacework access token"
+  dockerhub-user:
     required: true
-    description: Docker image tag
-  image-platform:
-    description: Target platform to build image for (eg. linux/amd64 (default), linux/arm64, etc)
-    required: false
-    default: linux/amd64
-
-  build-args:
-    required: false
-    description: List of build arguments for docker build as key-value pairs (e.g., KEY=VALUE)
-    default: ""
-  secrets:
```

Comment:
This would also be a breaking change. I think we should keep this input and allow to pass it through to the docker build action